packages/cosmic-swingset/lib/ag-solo/vats/repl.js

```diff
@@ -8,7 +9,6 @@ import makeUIAgentMakers from './ui-agent';
 export function stringify(
   value,
   spaces,
-  getInterfaceOf,
```

Comment:
Why make an incompatible change to the API?  Just rename the argument to avoid the shadowing, and default its value as in:

```js
  gio = getInterfaceOf,
```

and then use `gio` within the stringify function.

Reply:
Aye. I generally like to delete code when it's no longer needed, but I agree it's quicker and only slightly ugly to leave this argument in place.